### PR TITLE
add branch lesson2

### PR DESCRIPTION
**Pull request by yourMrDruid**

### CODE REVIEW

File: src/Lesson2/Lesson.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package Lesson2;
+
+public class Lesson {
+    public static void main(String[] args) {
```
Comment (yourMrDruid) [inline src/Lesson2/Lesson.java:4]: 4 line

File: src/Lesson2/Lesson.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package Lesson2;
+
+public class Lesson {
```
Comment (milyjdrug84) [inline src/Lesson2/Lesson.java:3]: Line 3

File: src/Lesson2/Lesson.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    public static void main(String[] args) {
+        System.out.println("as");
+    }
+}
```
Comment (milyjdrug84) [inline src/Lesson2/Lesson.java:7]: Line 7

**Review by milyjdrug84 — CHANGES_REQUESTED** [review]
any

**Pull request merged**